README.md

```diff
@@ -1,98 +1,173 @@
+usagereports.rooturl = http://localhost:8082
 ```
-You can try one of the variety of [quickstart
-guides](https://docs.docker.com/get-started/part2/) to make sure that
-your setup is otherwise working.
 
-This configuration uses [`docker-compose`](https://github.com/docker/compose) and requires at least version `1.13`. If the version in your OS repo is too old then the latest binaries can be found at https://github.com/docker/compose/releases.
+This will require a restart of Mantid.
+Note that the port number depends on the value specified by `HOST_PORT`
+specified in the environment file but defaults to the above value.
+See [DevelopmentSetup](DevelopmentSetup.md#creating-an-environment-env-file)
+for how to setup and environment file.
+
+## Getting Started
 
+To get started for development please follow the instructions in [DevelopmentSetup](DevelopmentSetup.md).
+
+<!--
 Much of the following is heavily adapted from the [docker django instructions](https://docs.docker.com/compose/django/) and
-https://realpython.com/blog/python/django-development-with-docker-compose-and-machine/, which uses an example repo at
-https://github.com/realpython/dockerizing-django.
+<https://realpython.com/blog/python/django-development-with-docker-compose-and-machine/>, which uses an example repo at
+<https://github.com/realpython/dockerizing-django>.
+
+To start the services locally you will first need to create a `.env` file next to `docker-compose.yml` containing, for example:
+
+        DB_NAME=Test
+        DB_USER=YourName
+        DB_PASS=APassWord
+        HOST_PORT=8082
+        EMAIL_HOST=smtp.sparkpostmail.com
+        EMAIL_HOST_USER=SMTP_Injection
+        EMAIL_PORT=587
+        EMAIL_HOST_PASSWORD=<Api password need to retrieve from sparkpost>
+        EMAIL_TO_ADDRESS=<email to recieve reports>
+        EMAIL_FROM_ADDRESS=error-reports@mantidproject.org
+        SLACK_WEBHOOK_URL=https://hooks.slack.com/services/.......
 
-To start the services locally you will first need to create a `.env` file next to `docker-compose.yml` containing the the following information:
-* `DB_NAME`
-* `DB_USER`
-* `DB_PASS`
-* `HOST_PORT`
+The values of each key are irrelevant for the test setup. For production they need to be kept secret. The email enviromental variables may be left out if emailing functionality is not required.
 
 Now start the services with:
 
 ```
-$ bin/boot.sh
+bin/boot.sh
 ```
-and the site will be viewable at `localhost:8082`.
+
+and the site will be viewable at `http://localhost:8082/admin`.
+
+To stop the services execute:
+
+```
+bin/shutdown.sh
+```
+
+which ensures that the webdata volume is cleaned up.
+
+Working with Django
+-------------------
+
+The first time you create a database you will need to create a Django admin account to access the Django admin interface. To do so first get into the docker container by running `docker exec -it <docker-web-container-name> bash` and then run `python manage.py createsuperuser`.
```

Comment:
```suggestion
`docker exec -it <docker-web-container-name> bash` and then run `python manage.py createsuperuser`.
```

Can we keep some of this superuser stuff in either the README or DevSetup?

Reply:
Yes. Will do. I commented this old stuff out as I didn't have everything ready in time for Monday.

I wonder if we can do this automatically when running django for the first time.